src/igmpproxy.h

```diff
@@ -60,6 +62,11 @@
 #include <netinet/in.h>
 #include <arpa/inet.h>
 
+#ifdef __FreeBSD__
+    #include "config.h"
```

Comment:
Same here, config.h must be included as the first file.

Reply:
issue #35